src/Installers/Windows/Wix.targets

```
@@ -2,7 +2,7 @@
   <!-- Set versioning properties after Arcade SDK targets have been imported. -->
   <PropertyGroup>
     <!-- Used for generating stable upgrade codes for bundles -->
-    <Version>$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion).$(AspNetCorePatchVersion).0</Version>
+    <Version>$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion).$(AspNetCorePatchVersion).$(FileRevisionVersion)</Version>
```

Weird, I thought I shortened all `$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion)` instances. If we need this,
```suggestion
    <Version>$(AspNetCoreMajorMinorVersion).$(AspNetCorePatchVersion).$(FileRevisionVersion)</Version>
```

---

@joeloff is this the right fix for 5.0❔